### ISSUE TITLE
Max video lenght

### ISSUE THREAD

**Issue by jansonz** [kaltura/nginx-vod-module#1260]
Hi, We have been trying to use the origin on a 8.5 hour long asset and seems that a response is never generated. 
Does the sw read the whole file before manifest is generated or it relays on moov and the configuration set in vod_max_metadata_size?

Do you know what is the max length asset the sw has been tested with?

Thank you,

**Comment by erankor** [comment]
No, the module does not read the whole file, it reads only the moov atom. 
I don't remember an exact maximum video duration that I tested with, but the most common problem with long videos is when the configuration has `vod_manifest_segment_durations_mode accurate`, in this case the module tries to load the metadata of all the frames in memory, and at some point hits the limit of 1m frames per request that is set in the code.
Did you check the error log? are there any errors there?

**Comment by jansonz** [comment]
Thank you for the feedback. We indeed are using `vod_manifest_segment_durations_mode accurate` so then this could make sense. 
Need to re-check the logs but don't recall seeing anything in particular.
